File: Database/src/main/java/Database/DTOs/IssueCreationDTO.java
package Database.DTOs;

public class IssueCreationDTO {
    private String title;
    private String body;
    private int ownerId;

    public IssueCreationDTO(String title, String body, int ownerId) {
        this.title = title;
        this.body = body;
        this.ownerId = ownerId;
    }

    public String getTitle() {
        return title;
    }

    public String getBody() {
        return body;
    }

    public int getOwnerId() {
        return ownerId;
    }
}

File: Website/Shared/DTOs/IssueCreationDto.cs
namespace Shared.DTOs;

public class IssueCreationDto
{
    public string Title {get; set; }

    public string Body { get; set; }

    public int Owner { get; set; }
    public IssueCreationDto(string title, string body, int owner)
    {
        Title = title;
        Body = body;
        Owner = owner;
    }
}